lib/rootstrap/tasks/add_code_analysis.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Rootstrap
+  module Tasks
+    class AddCodeAnalysis
+      attr_reader :context
+
+      def initialize(context:)
+        @context = context
+      end
+
+      def add
+        write_to_rakefile
+      end
+
+      private
+
+      def rakefile
+        @rakefile ||= File.read("#{File.dirname(__FILE__)}/../support/Rakefile")
+      end
+
+      def write_to_rakefile
```

Comment:
why is this private and you are accessing it like an alias method 🤔 ?

Reply:
If you really want to do an alias (or call methods different)I found this: https://stackoverflow.com/questions/31983479/alias-method-a-private-method that shows some options

Reply:
Good point. I was just following the structure of the other tasks, but it's true that I can move the body of this method to the main one